src/overlays/actors/ovl_En_Hg/z_en_hg.c

```diff
@@ -184,92 +199,93 @@ void func_80BCF4AC(EnHg* this, PlayState* play) {
         }
         if ((Math_Vec3f_DistXZ(&this->actor.world.pos, &this->actor.home.pos) > 200.0f) &&
             (Math_Vec3f_DistXZ(&player->actor.world.pos, &this->actor.home.pos) > 200.0f)) {
-            func_80BCF5F0(this);
+            EnHg_SetupDetectPlayerDistanceFromHomePos(this);
         }
     }
 }
 
-void func_80BCF5F0(EnHg* this) {
+void EnHg_SetupDetectPlayerDistanceFromHomePos(EnHg* this) {
```

Comment:
Hmm, this seems like another wait function if Link gets to far when he is chasing him. Perhaps something like `EnHg_MoveTowardsPlayerWait` or `EnHg_ChasePlayerWait`?